files/id-stg/docker-compose.yml

```diff
@@ -1,40 +1,40 @@
-version: "3"
+version: '3'
+
 services:
-  mongodb:
-    restart: "always"
-    image: mongo:3.4.9
-    healthcheck:
-      test: ["CMD", "mongo", "--quiet", "localhost/test", "--eval", "'quit(db.runCommand({ ping: 1 }).ok ? 0 : 1)';"]
-      interval: 1m30s
-      timeout: 10s
-      retries: 3
+  postgres:
```

Comment:
Please add a healthcheck for postgres. See e.g. https://github.com/OpenConceptLab/oclsso/blob/main/docker-compose.yml#L10-L11